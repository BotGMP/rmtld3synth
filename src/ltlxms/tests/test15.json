[ "Always", 
  [ "Or", 
    [ "Overlap", [ "Expand", [ "Proposition", "a" ], 1.0 ], [ "Proposition", "b" ] ],
    [ "Eventually", [ "SubSetEq", [ "Proposition", "a" ], [ "Proposition", "b" ] ] ]
  ]
]