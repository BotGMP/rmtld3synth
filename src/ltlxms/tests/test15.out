(let ((a!1 (or (overlap (expand a 1.0) b)
               (eventually (forall ((xs Real) (ys Real)) (=> a b))))))
  (always a!1))
